savant/client/__init__.py

```diff
@@ -1,17 +1,27 @@
 """Source/Sink framework for development and QA purposes."""
-
 from .builder.sink import SinkBuilder
 from .builder.source import SourceBuilder
-from .frame_source import FrameSource, JpegSource, PngSource
+from .frame_source import FrameSource
 from .log_provider import LogProvider
 from .log_provider.jaeger import JaegerLogProvider
 
 __all__ = [
     'SinkBuilder',
     'SourceBuilder',
-    'JpegSource',
-    'PngSource',
     'LogProvider',
     'FrameSource',
     'JaegerLogProvider',
 ]
+
+try:
```

Comment:
Why not just add `python-magic` to requirements?

Reply:
I assumed this was the same situation as before with opencv dependency.

Let me simplify, and add the package to adapter requirements.